.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,21 @@
-Please delete this line and the text below before submitting your contribution.
+<details>
+<summary><strong>Instructions</strong></summary>
 
----
+Thanks for contributing! :heart:
 
-Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  
+If this contribution is for instructor training, please email link to this contribution to
+checkout@carpentries.org so we can record your progress. You've completed your contribution
+step for instructor checkout by submitting this contribution!
 
 If this issue is about a specific episode within a lesson, please provide its link or filename.
 
-Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  
+Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
+respond to your contribution. Although not all contributions can be incorporated into the lesson
+materials, we appreciate your time and effort to improve the curriculum. If you have any questions
+about the lesson maintenance process or would like to volunteer your time as a contribution
+reviewer, please contact The Carpentries Team at team@carpentries.org.
 
----
+You may delete these instructions from your comment.
```

Comment:
should we indicate everything between the <detail> tags? or is this sufficient as is?

Reply:
I think this is sufficient because sometimes people ignore even "Please delete..." request. But I'm open to suggestions.